addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/test/EmulatedV6Bridge.java

```diff
@@ -147,7 +147,7 @@ public void runDiscovery() {
             if (willbeclosed) {
                 return;
             }
-            logger.error(e.getLocalizedMessage());
+            logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace